assets/js/blocks/collection-filters/inner-blocks/stock-filter/utils.ts

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Some unit test here should be nice, I have to do the same for price and attribute.

Reply:
This was all copypasta actually. None of these are used so I will delete this.